src/Microsoft.Health.Dicom.Api/Registration/DicomServerServiceCollectionExtensions.cs

```diff
@@ -94,6 +95,14 @@ public static IDicomServerBuilder AddBackgroundWorkers(this IDicomServerBuilder
                 jsonOptions.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
             });
 
+            services.AddApiVersioning(c =>
+            {
+                c.AssumeDefaultVersionWhenUnspecified = true;
+                c.DefaultApiVersion = new ApiVersion(1, 0, "prerelease");
+                c.ReportApiVersions = true;
```

Comment:
We'll probably want to remember to document this new header we'll return

Reply:
Good call. Where should this documentation go? I didn't see any obvious place in the docs folder

Reply:
I imagine when this feature is done we'll have a doc like `api-versioning.md` or something where we'll talk about the URLs, how long a feature is supported ,etc